nexus_keycode/protocols/passthrough_uart.py

```diff
@@ -0,0 +1,27 @@
+import siphash
+
+NEXUS_INTEGRITY_CHECK_FIXED_00_KEY = "\x00" * 16
```

Comment:
To make both Python2 and Python3 happy here, you can make this change:

```
NEXUS_INTEGRITY_CHECK_FIXED_00_KEY = b"\x00" * 16
```

(notice the 'b' in front, indicating 'bytestring')